daemon/algod/api/server/v2/account.go

```diff
@@ -128,10 +126,14 @@ func AccountDataToAccount(
 		RewardBase:                  &record.RewardsBase,
 		Participation:               apiParticipation,
 		CreatedAssets:               &createdAssets,
+		TotalCreatedAssets:          uint64(len(createdAssets)),
 		CreatedApps:                 &createdApps,
+		TotalCreatedApps:            uint64(len(createdApps)),
 		Assets:                      &assets,
+		TotalAssets:                 uint64(len(assets)),
 		AuthAddr:                    addrOrNil(record.AuthAddr),
 		AppsLocalState:              &appsLocalState,
+		TotalAppsLocalState:         uint64(len(appsLocalState)),
```

Comment:
Why are we adding the Total* forms? I think it might be to let the caller calculate minimum balance requirements? If so, that seems unneeded since we give them that info. Further, I don't think it's enough, unless AppsTotalSchema included global schemas (for apps created by this account) _and_ local schemas (for apps this account has opted into) but I think maybe it's just the globals?

Reply:
The caller can use this information to figure out how many resources are associated with this object.

Reply:
Maybe `TotalAssets` -> `TotalAssetsOptedIn` and `TotalLocalStates` -> `TotalAppsOptedIn`?

Reply:
@cce making this change would require to change the oas file. Could you take a look to figure out its implications ?

Reply:
OK from discussion just now with @yaovi-a and @tsachiherman it is more clear to call them TotalAssetsOptedIn and TotalAppsOptedIn, plus similar names in the REST API fields. will update ASAP @tolikzinovyev @nicholasguoalgorand

Reply:
Updated in https://github.com/algorand/go-algorand/pull/3676